manticore/utils/log.py

```diff
@@ -100,11 +100,12 @@ def get_levels():
             ("manticore.platforms.*", logging.DEBUG),
             ("manticore.ethereum", logging.DEBUG),
             ("manticore.core.plugin", logging.DEBUG),
+            ("manticore.wasm.*", logging.INFO),
```

Comment:
Should there be something about wasm in the first level, labeled `#1` above?

Reply:
By default, that'll be `logging.warning`, which should be fine. I use logging.info for printing WASM instructions and logging.debug for printing stack operations, so there's not a lot of middle ground for purely informational messages. I guess I could maybe make a logging.verbose level?

Reply:
`logging.warning` is fine. I was just wondering because there seemed to be a pattern with enabling `logging.info` in level 1 for the different languages
```
        # 1
        [...
         ("manticore.ethereum.*", logging.INFO),
         ("manticore.native.*", logging.INFO),
         ...
         ]
```